src/util/loanSearch/searchStateUtils.js

```diff
@@ -73,3 +74,20 @@ export async function updateSearchState(apollo, loanQueryFilters, allFacets, que
 		}
 	});
 }
+
+/**
+ * Creates a saved search with a name and set of filters
+ * @param {Object} apollo The Apollo client instance
+ * @param {Object} loanQueryFilters The filters for the saved search
+ * @param {string} savedSearchName The name of the saved search
+ * @returns {Promise<Array>} Promise for the results of the mutation
+ */
+export async function createSavedSearch(apollo, loanQueryFilters, savedSearchName) {
```

Comment:
I'd recommend throwing this in a new util file `savedSearchUtils` that is dedicated to saved search within loan filtering. This file is currently focused on the search state in particular, not saved search.

Reply:
If you do create a new file, please update the `README` in this folder with a brief description of the new util.